### PR TITLE
Clean up dangling debug code

### PR DESCRIPTION
**Pull request by vicmgs**

### CODE REVIEW

File: assets/js/components/labels/LabelShow.jsx [hunk 1]
```diff
@@ -126,24 +126,17 @@ class LabelShow extends Component {
     this.props.updateLabelNotificationWebhooks(webhooks);
   }
 
-  handleToggleDebug = () => {
-    const { showDebugSidebar } = this.state
-
-    if (!showDebugSidebar) {
-      this.props.toggleLabelDebug(this.props.match.params.id)
-      analyticsLogger.logEvent("ACTION_OPEN_LABEL_DEBUG", { "id": this.props.match.params.id })
```
Comment (ambuznego) [inline assets/js/components/labels/LabelShow.jsx:134]: wouldn't we wanna keep the analytics log?

File: assets/js/components/devices/DeviceShow.jsx [hunk 2]
```diff
@@ -176,18 +181,6 @@ class DeviceShow extends Component {
     this.setState({ showDeleteDeviceModal: false })
   }
 
-  handleToggleDebug = () => {
-    const { showDebugSidebar } = this.state
-
-    if (!showDebugSidebar) {
-      this.props.toggleDeviceDebug(this.props.match.params.id)
-      analyticsLogger.logEvent("ACTION_OPEN_DEVICE_DEBUG", { "id": this.props.match.params.id })
```
Comment (ambuznego) [inline assets/js/components/devices/DeviceShow.jsx:184]: same here

**Review by ambuznego — APPROVED** [review]
one minor comment, thanks for cleaning this up 🙏

File: assets/js/components/labels/LabelShow.jsx (hunk 1 above)
Comment (vicmgs) [reply assets/js/components/labels/LabelShow.jsx:134]: good catch ty

**Pull request merged**